fix for https://github.com/wso2/docs-apim/issues/1129

Mariangela opened the pull request:
## Purpose
fix for https://github.com/wso2/docs-apim/issues/1129

shaniR commented on en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:
```diff
@@ -0,0 +1,215 @@
+# Configuring Keystores in API-M Analytics
+
+WSO2 products use asymmetric cryptography by default for authentication and data encryption. In asymmetric cryptography, keystores (with key pairs and certificates) are created and stored for the product. Keystore is a repository where private keys and certificates can be stored. It is possible to have multiple keystores so that the keys used for different use cases are kept unique. For more info ...
```
It is possible to have multiple keystores so that the keys used for different use cases are kept unique -> shouldn't this be "It is possible to have multiple keystores so that the keys used for different use cases are kept seperately"

Mariangela replied:
Fixed

shaniR commented on en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:
```diff
@@ -0,0 +1,215 @@
+!!! note
+    Other than explicitly mentioned, you need to do all the following configuration updates related to the dashboard profile in the `<API-M_ANALYTICS_HOME>/conf/dashboard/deployment.yaml` file.
+
+1. Update the Databridge configurations.
```
should Databrige be in camel case?

Mariangela replied:
@fazlan-nazeem , please clarify.

fazlan-nazeem replied:
DataBridge should be fine

Mariangela replied:
Fixed

shaniR commented on en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:
```diff
@@ -0,0 +1,215 @@
+
+1. Update the Databridge configurations.
+
+    This is used for authenticating communication over SSL/TLS between API Manager and API-M Analytics. Handles thrift and binary transports.  
```
Authenticating the communication

Mariangela replied:
Fixed

shaniR commented on en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:
```diff
@@ -0,0 +1,215 @@
+       
+    | Element                | Description                                                      |
+    |------------------------|------------------------------------------------------------------|
+    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
```
should we mention about the formats supported as well?

Mariangela replied:
@fazlan-nazeem , please share your views.

fazlan-nazeem replied:
do you mean other keystore file formats?

Mariangela replied:
@shaniR , please clarify @fazlan-nazeem 's Q

bhathiya replied:
we support .jks only.

Mariangela replied:
fixed

shaniR commented on en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md:
```diff
@@ -0,0 +1,215 @@
+    2. Provide the keystore and private key password in the Base64 encoded format in the `<ANALYTICS_HOME>/conf/dashboard/master-keys.yaml` file.
+
+        ??? info "Click here for more info on the `master-keys.yaml` file"
+            The base64 encoded plaintext keystore password and the base64 encoded plaintext private password of the keystore used for Secure Vault is defined in the master-keys.yaml file. WSO2 Carbon Server reads this file to obtain these two passwords, so that it can decrypt the passwords in the `secrets.properties` file in order to start the server.
```
plain text -> two words?

shaniR replied:
Shall we reword this to, "The keystore password and private password which is in plain text is base64 encoded and stored in the master-keys.yaml"

Mariangela replied:
Fixed.
Changed to 
"The keystore password and private password used for Secure Vault, which is in plain text is base64 encoded and stored in the `master-keys.yaml` file. "

Mariangela commented:
@shaniR , I have addressed all the comments that you have mentioned other that the comment where Fazlan had raised a Q on top of. Pls check and approve.

Mariangela commented:
@shaniR , I updated this further. Please review and merge.
